appsec/src/extension/ip_extraction.c

```diff
@@ -17,6 +17,7 @@
 #include <arpa/inet.h>
```

Comment:
We manually keep this script aligned with `ext/ip_extraction.c` so I think you should add this new header there  as well

Reply:
hm is there any reason to have this duplicated rather than in only one place?

Reply:
IIRC when we implemented this we were still in separate repositories and APM needs to collect the client IP using the same strategy when ASM is disabled but `DD_TRACE_CLIENT_IP_ENABLED` is set to true. Feel free to change it to a common implementation.